s3-import-commons/src/main/java/no/unit/nva/publication/s3imports/FilenameEventEmitter.java

```diff
@@ -2,6 +2,7 @@
 
 import static java.util.Objects.isNull;
 import static no.unit.nva.publication.s3imports.ApplicationConstants.ERRORS_FOLDER;
+import static no.unit.nva.publication.s3imports.ApplicationConstants.defaultClock;
```

Comment:
FilenameEventEmitter produces errors as well and its error paths should be updated too.